test/run-spec-wasm2c.py

```diff
@@ -105,6 +105,18 @@ def Mangle(match):
     return 'Z_' + re.sub(pattern, Mangle, s.encode('utf-8')).decode('utf-8')
 
 
+def MangleCWriterName(wasm_filename):
```

Comment:
How is this different to `MangleName` just above?   Why do we need to ways to mangle a name?

Reply:
This was not being called -- it was apparently vestigial. I deleted it.